nemo_skills/evaluation/metrics/audio_metrics.py

```
@@ -0,0 +1,371 @@
+    def get_metrics(self):
+        """Get computed metrics.
+        
+        Aggregates all collected metric scores and computes averages.
+        Converts error rates and scores to percentages for reporting.
+        
+        Returns:
+            Dictionary of aggregated metrics by evaluation mode
+        """
+        metrics_dict = super().get_metrics()
+
+        for agg_mode, agg_metrics in metrics_dict.items():
```

Ruff (0.14.8)

Loop control variable `agg_mode` not used within loop body

Rename unused `agg_mode` to `_agg_mode`

(B007)